parser/model/model.go

```diff
@@ -1163,8 +1163,11 @@ type PartitionInfo struct {
 	AddingDefinitions []PartitionDefinition `json:"adding_definitions"`
 	// DroppingDefinitions is filled when dropping a partition that is in the mid state.
 	DroppingDefinitions []PartitionDefinition `json:"dropping_definitions"`
-	States              []PartitionState      `json:"states"`
-	Num                 uint64                `json:"num"`
+	// OriginalDefinitions is filled when truncating partitions that is in the mid state.
+	OriginalDefinitions []PartitionDefinition `json:"new_definitions"`
```

Comment:
I considered it, but maybe not graceful. 
Not only the dropping definitions is used in other circumstances,  but I have to store truncated definitions' positions in the old definitions, so that I can joint the original definitions when deleting global index.

Reply:
Why would you need the positions in the original definitions, is not Physical Table IDs the only thing that is needed from each truncated partition?

Reply:
@mjonss OK, I understand what you means. I reuse the DroppingDefinitions and remove the OriginalDefinitions.